ban/core/models.py

```diff
@@ -117,6 +117,7 @@ def housenumbers(self):
 
 
 class HouseNumber(Model):
+    CEA_FORMAT = '[\dAB]{2}\d{3}[234679ABCEGHILMNPRSTUVXYZ]{5}'
```

Comment:
Thanks, added a comment. :)
But then it means I can send a mail to someone just puting the CEA in place of the address? :)